game/src/main/java/org/apollo/game/model/entity/Entity.java

```diff
@@ -59,6 +61,13 @@ public World getWorld() {
 	 */
 	public abstract EntityType getEntityType();
 
+	/**
+	 * Returns the set of positions that this Entity occupies.
+	 *
+	 * @return The set of occupied positions.
+	 */
+	public abstract Set<Position> getBounds();
```

Comment:
Let me know if you'd like to change the name of this, if you can think of a better name. It might be confusing with `NPC#getBoundries()` already there.

Reply:
I think it'd be better to return an object representing the rectangle of the bounds..  Recomputing every position in the `Mob`s bounds is going to result in a ton of temporary objects with a lot of NPCs active.  

A rectangle is also going to result in faster `contains` checks.